fuzz/fuzz_layout.cpp

```diff
@@ -0,0 +1,50 @@
+#include <fuzzer/FuzzedDataProvider.h>
+#include <yoga/Yoga.h>
+#include <cstdint>
+
+YGFlexDirection fuzzed_flex_direction(FuzzedDataProvider& fdp) {
+  return fdp.PickValueInArray({
+      YGFlexDirectionColumn,
+      YGFlexDirectionColumnReverse,
+      YGFlexDirectionRow,
+      YGFlexDirectionRowReverse,
+  });
+}
+
+void FillFuzzedTree(
+    FuzzedDataProvider& fdp,
+    const YGConfigRef& config,
+    const YGNodeRef& root,
```

Comment:
I missed this the first time around, but `YGConfigRef` and `YGNodeRef` are themselves just pointers. That also makes `const` apply to the pointer value, instead of the underlying node/config.

Needed to fix a quick license header to make internal linter happy, but I'm going to make a quick ninja-edit to make that clearer.

Reply:
Same for a couple other quick repo conventions around naming

Reply:
No worries. Let me know if there is anything you'd like me to do here :)